pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -132,7 +128,8 @@ func (q *QueryReconciler) runShowRestorePoints(ctx context.Context, dispatcher v
 	opts []showrestorepoints.Option) (err error) {
 	// set Querying status condition ,state and restore points prior to calling vclusterops API
 	err = vrpqstatus.Update(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
-		[]*metav1.Condition{v1.MakeCondition(vapi.Querying, metav1.ConditionTrue, "Started")}, stateQuerying, nil)
+		[]*metav1.Condition{v1.MakeCondition(vapi.QueryReady, metav1.ConditionTrue, "Started"),
```

Comment:
We shouldn't be touching QueryReady in this reconciler. That should only be modified in the proceeding reconciler.